src/common/containers/landing.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+import { Anchor } from '../components/button';
+
+import styles from './container.css';
+
+class Landing extends Component {
+  componentWillMount() {
+    if (this.props.auth.authenticated) {
```

Comment:
If landing container detects authenticated user it redirects to dashboard (`/repos`) route.

So authenticated user will not see landing page (they will always get to dashboard), but dashboard still has separate URL.

This probably means that we also need to verify if user is authenticated on `/repos` route, so that not authenticated user would not be able to visit it (redirect back to landing? or just go to GH login directly?).

Other way to solve this would be to keep both landing and dashboard in `/` route (same container), but during render we would detect if user authenticated or not and render different component (landing page contents or user dashboard).

@cjwatson @katiefenn what do you think?